src/GraphQL/Queries/Concerns/FiltersQuery.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Statamic\GraphQL\Queries\Concerns;
+
+use Statamic\Support\Arr;
+
+trait FiltersQuery
+{
```

Comment:
Can you add `use QueriesConditions` in here?

Since this trait relies on it, we'll remove it from the query classes and use it here.